### PR TITLE
Refactor token use api

### PR DESCRIPTION
**Pull request by Opeyem1a**
Converting TokenUse api to use a class-based approach

### CODE REVIEW

File: api/views/token_use.py [hunk 1]
```diff
@@ -1,37 +1,25 @@
-import re
-
 from django.shortcuts import get_object_or_404
```
Comment (keyvankhademi) [inline api/views/token_use.py:1]: you imported the wrong get_object_or_404, please use the rest_framework version for APIs.

File: api/serializers/course.py [hunk 2]
```diff
@@ -1,7 +1,8 @@
 from rest_framework import serializers
 
 from accounts.models import MyAnonymousUser
-from api.serializers import QuestionSerializer, UQJSerializer
+from api.serializers.question import QuestionSerializer
```
Comment (keyvankhademi) [inline api/serializers/course.py:4]: Either fix all of them or leave them as they are. also the same for __init__.py s. I would recommend to fix them all if you want :D

Comment (Opeyem1a) [reply api/serializers/course.py:4]: fixing as in making it so every import come from the specific file it references?

Comment (keyvankhademi) [reply api/serializers/course.py]: No, its importing all the serializers in __init__.py and having one line of import here.

File: api/views/action.py [hunk 3]
```diff
@@ -2,7 +2,7 @@
 from rest_framework.permissions import IsAuthenticated
 
 from api.pagination import BasePagination
-from api.serializers import ActionsSerializer
+from api.serializers.action import ActionsSerializer
```
Comment (keyvankhademi) [inline api/views/action.py:5]: I think this should be rolled back to what it was.

**Review by keyvankhademi — APPROVED** [review]

**Pull request merged**